src/sql/src/func.rs

```diff
@@ -3552,11 +3552,9 @@ pub static MZ_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
                 CASE
                 -- We need to validate the name and privileges to return a proper error before
                 -- anything else.
-                WHEN mz_unsafe.mz_error_if_null(
-                    (SELECT name FROM mz_clusters WHERE name = $2),
-                    'error cluster \"' || $2 || '\" does not exist'
-                ) IS NULL
-                OR NOT mz_internal.mz_validate_privileges($3)
+                WHEN NOT EXISTS (SELECT name FROM mz_clusters WHERE name = $2)
+                THEN mz_unsafe.mz_error_if_null(NULL, 'error cluster \"' || $2 || '\" does not exist')
```

Comment:
I think this will fix your CI failures.

```suggestion
                THEN mz_unsafe.mz_error_if_null(NULL::boolean, 'error cluster \"' || $2 || '\" does not exist')
```